### PR TITLE
Add financing fee in ledger credit lines

### PR DESCRIPTION
**Pull request by ankur-singh-dehaat**
# Description
1. Add Financing Fee in Transactions
2. Resolve Ui issues
3. Upgrade accompanist library

### CODE REVIEW

File: lib-ledger/src/main/java/lib/dehaat/ledger/presentation/ledger/ui/component/FilterHeader.kt [hunk 1]
```diff
@@ -47,20 +56,32 @@ fun FilterHeader(
 
     Spacer(modifier = Modifier.width(16.dp))
 
+    val title = buildAnnotatedString {
+        withStyle(spanParagraphT2Highlight(Neutral80)) {
```
Comment (lovemhjn) [inline lib-ledger/src/main/java/lib/dehaat/ledger/presentation/ledger/ui/component/FilterHeader.kt:60]: Put this in a separate function

**Review by lovemhjn — CHANGES_REQUESTED** [review]

File: lib-ledger/src/main/java/lib/dehaat/ledger/presentation/ledger/ui/component/FilterHeader.kt (hunk 1 above)
Comment (ankur-singh-dehaat) [reply lib-ledger/src/main/java/lib/dehaat/ledger/presentation/ledger/ui/component/FilterHeader.kt]: addressed

**Review by lovemhjn — APPROVED** [review]
lgtm

**Review by lovemhjn — APPROVED** [review]
lgtm

**Pull request merged**